lib/measures/tbd/resources/ua.rb

```diff
@@ -341,13 +341,18 @@ def uprate(model = nil, s = {}, argh = {})
 
             if sss.isConstructionDefaulted
               set = defaultConstructionSet(sss) # building? story?
-              constructions = set.defaultExteriorSurfaceConstructions
 
-              unless constructions.empty?
-                constructions = constructions.get
-                constructions.setWallConstruction(lc)        if typ == :wall
-                constructions.setFloorConstruction(lc)       if typ == :floor
-                constructions.setRoofCeilingConstruction(lc) if typ == :ceiling
+              if set.nil?
```

Comment:
_Osut_'s `defaultConstructionSet` failed to retrieve the applicable set for a surface's construction, as the method wasn't (also) looking for an inherited construction from its adjacent surface - not good. The _OSut_ method has been fixed/tested. Adding here an extra fallback (just in case). This still needs a dedicated stress test.